monero_utils/MyMoneroCoreBridgeEssentialsClass.js

```diff
@@ -313,7 +312,7 @@ class MyMoneroCoreBridgeEssentialsClass extends MyMoneroBridgeClass_Base
 		{
 			fee_per_b: typeof optl__fee_per_b_string !== undefined && optl__fee_per_b_string != null 
 				? optl__fee_per_b_string 
-				: (new JSBigInt(fee_per_kb__string)).divide(1024).toString()/*kib -> b*/, 
+				: `${BigInt(fee_per_kb__string) / BigInt(1024)}` /*kib -> b*/,
```

Comment:
It's not supported on iOS 13.

It's not used so we can drop this function entirely.